Source/Particles/PhysicalParticleContainer.H

```diff
@@ -174,6 +174,16 @@ public:
     // Inject particles in Box 'part_box'
     virtual void AddParticles (int lev);
 
+    /**
```

Comment:
It would be better if this change was made in a separate PR since it is not directly related to this change.

Reply:
You're right @dpgrote , thanks for pointing this out.
I will move it to another PR!